lib/CZip.h

```diff
@@ -19,6 +19,11 @@
 #ifndef __CZIPFILE_H_
 #define __CZIPFILE_H_
 
+#include <cstdio>
+#ifdef __LINUX__
+#include "linux/linux_fix.h"
```

Comment:
Do we use anything from this include in this file?

Reply:
The files that include `CZip.h` both require `_stat`.

Reply:
IMO it would make sense to include linux_fix in those files to follow the include what you use philosophy